test/exoscale/compute/api/payload_test.clj

```diff
@@ -0,0 +1,48 @@
+(ns exoscale.compute.api.payload-test
+  (:require [exoscale.compute.api.payload :refer [sign build-payload]]
+            [clojure.test :refer :all]
+            [clj-time.core :as t]))
+
+(def API_KEY "key")
+(def API_SECRET "secret")
+
+;; NOTE: the following test has been borrowed from clostack:
+;; https://github.com/exoscale/clostack/blob/master/test/clostack/payload_test.clj
+;; It doesn't work here as we've got changes in the signing code.
+;; What should be done here is, we need to decide which signing
```

Comment:
TODO as part of this PR or a separate one? I'm not sure we should commit a commented out test

Reply:
Good point, I'll drop that and create a ticket then.